projects/sunbird-player-sdk/src/lib/player-utils/service/content-compability/content-compability.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ContentCompabilityService {
+
+  constructor() { }
+
```

Comment:
**defaultCompatibilityLevel** to be rename to **playerContentCompatibiltyLevel** and value should be added here only